app/src/main/AndroidManifest.xml

```diff
@@ -7,7 +7,7 @@
     <application
         android:name=".MifosSelfServiceApp"
         android:allowBackup="true"
-        android:icon="@mipmap/ic_launcher"
+        android:icon="@drawable/ic_icon"
```

Comment:
I have added by typing the command `git add` , and also checked the status and it showed that it added the png file, but here I am not getting the image